PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,13 @@
 <!---
-[ If your PR is ready for review ] 
+[ If your PR is ready for review ]
 please apply the "PR needs review" label to it
 
-[ otherwise ] 
+[ otherwise ]
 please put "WIP" in the PR title and apply the "PR needs work" label. Once PR is ready for review, remember to remove "WIP" from the PR title and replace "PR needs work" label with "PR needs review"
 -->
 
+- [ ] Localization addressed
+
```

Comment:
Do we at least want to let them know what they need to make sure happened? or point them to the README/wiki?